src/microservices/Microservices.DicomReprocessor/DicomReprocessorControlMessageHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+using System;
+using JetBrains.Annotations;
+using NLog;
+using Smi.Common.Options;
+
+
+namespace Microservices.DicomReprocessor
+{
+    public class DicomReprocessorControlMessageHandler
```

Comment:
Just a thought :

Can't we just make this `IControlHandler` with a method signature for `void ControlMessageHandler(string, string)` that way we can avoid Action<string,string> and make it easier to find them all etc?

Reply:
That definitely makes sense, which makes me wonder why I implemented it this way in the first place! Do we want to refactor that in this PR?

Reply:
Yeah it should be very small fix

Reply:
It's maybe something to do with the fact that not all services need specific control handlers, so they have to register a callback if they wish to handle any unique control messages.